README_dev.md

```diff
@@ -38,14 +46,16 @@ Mage.ai server uses at least Python 3.6 (as per `setup.py`), but the development
 As such, make sure you have Python >=3.8. Verify this with:
 
 ```bash
-python --version
+git clone https://github.com/mage-ai/mage-ai mage-ai
+cd mage-ai
 ```
 
 Using a virtual environment is recommended.
 
-<details>
-  <summary><b>Virtual environment guide</b></summary>
-#### Anaconda + Poetry
+## Configuring a Virtual Env
```

Comment:
It's a bit weird to separate "Configuring a Virtual Env" and "Installing Dependencies".
In the section "Anaconda + Poetry" under "Configuring a Virtual Env", it also includes the command of installing dependencies "make dev_env"

Reply:
At first I was going to remove the venv guide section entirely, then decided to keep it because maybe it'd help beginners, but presented it as a collapsible so whoever interested in the guide could see it and at the same time not making the whole file too long. Maybe @mattppal could tell us why you decided that it should be a separate section?

Reply:
@wangxiaoyou1993 should be a. bit more logical now